src/cluster.h

```diff
@@ -204,23 +204,23 @@ int clusterNodeTlsPort(clusterNode *node);
  *         │                              │◄────────────────────────┤                             │
  *         │                              │                         │ASM_EVENT_MIGRATE_WAIT_PAUSE │
  *         │                              │                         ├────────────────────────────►│
- *         │                              │                         │ ASM_REQUEST_MIGRATE_PAUSED  │
+ *         │                              │                         │ASM_REQ_MIGRATE_NOTIFY_PAUSED│
  *         │                              │                         │◄────────────────────────────┤
  *         │                              │ Drain repl stream       │                             │
  *         │                              │◄────────────────────────┤                             │
  *         │ASM_EVENT_IMPORT_WAIT_FINALIZE│                         │                             │
  *         │◄─────────────────────────────┤                         │                             │
  *         │                              │                         │                             │
- *         │ ASM_REQUEST_CONFIG_UPDATED   │                         │                             │
- *         ├─────────────────────────────►│                         │ ASM_REQUEST_CONFIG_UPDATED  │
+ *         │ ASM_REQ_NOTIFY_CONFIG_UPDATED│                         │                             │
+ *         ├─────────────────────────────►│                         │ASM_REQ_NOTIFY_CONFIG_UPDATED│
  *         │                              │                         │◄────────────────────────────┤
  *         │                              │                         │                             │
  */
 
-#define ASM_REQUEST_IMPORT_START        1  /* Start a new import operation (destination side) */
-#define ASM_REQUEST_IMPORT_CANCEL       2  /* Cancel an ongoing import operation (destination side) */
-#define ASM_REQUEST_MIGRATE_PAUSED      3  /* Notify that slot writes are paused (source side) */
-#define ASM_REQUEST_CONFIG_UPDATED      4  /* Notify that config is updated (source and destination side) */
+#define ASM_REQ_IMPORT_START          1  /* Start a new import operation (destination side) */
+#define ASM_REQ_IMPORT_CANCEL         2  /* Cancel an ongoing import operation (destination side) */
+#define ASM_REQ_MIGRATE_NOTIFY_PAUSED 3  /* Notify that slot writes are paused (source side) */
+#define ASM_REQ_NOTIFY_CONFIG_UPDATED 4  /* Notify that config is updated (source and destination side) */
```

Comment:
ai suggested to add NOTIFY_ to last two. I did it and to make these shorter REQUEST_ -> REQ_

Reply:
so REQ is necessary for the last two? they seem not request instead of notification
ASM_NOTIFY_MIGRATE_PAUSED
ASM_NOTIFY_CONFIG_UPDATED

Reply:
ok made the change